src/lib/styles/variables.less

```diff
@@ -3,7 +3,9 @@
 @grey-dark: #343434;
 @grey-dove: #676767;
 @grey: #818181;
+@grey-suva: #939393;
 @grey-silver: #a3a3a3;
+@grey-northernsky: #bebebe;
```

Comment:
This one is made up. It's rarely used, so it's got a reference so obscure you'll wonder why you're using it. BEBEBE is how Nick Drake tunes his guitar.